src/uda.rs

```diff
@@ -65,9 +65,11 @@ pub fn uda_schema() -> SubSchema {
                 GS_NOMINAL => Occurrences::Once,
                 GS_CONTRACT => Occurrences::Once,
                 GS_TIMESTAMP => Occurrences::Once,
+                GS_TOKENS => Occurrences::NoneOrOnce,
+                GS_ATTACH => Occurrences::NoneOrOnce,                   
             },
             assignments: tiny_bmap! {
-                OS_ASSET => Occurrences::Once,
+                OS_ASSET => Occurrences::OnceOrMore,
```

Comment:
No, we can't do that in that simple way :)
See my general review note.
```suggestion
                OS_ASSET => Occurrences::Once,
```